src/main/java/com/dougnoel/sentinel/elements/dropdowns/SelectElement.java

```diff
@@ -91,6 +91,15 @@ public String getText(int index){
         Select selectElement = new Select(this.element());
         return selectElement.getOptions().get(index).getText();
     }
+
+    /**
+     * Gets the text of the first item currently selected.
+     * @return String the text value of the selected option
+     */
+    @Override
+    public String getText(){
+        return getSelectedText();
+    }
```

Comment:
Add @deprecated to getSelectedText() if we don't need it anymore and we will remove it in a future version.

Reply:
Removed it altogether and refactored a chunk of Select text verification as a result.